lib/queuePopulator/IngestionProducer.js

```diff
@@ -123,7 +123,7 @@ class IngestionProducer {
     getRaftLog(raftId, begin, end, targetLeader, done) {
         const recordStream = new ListRecordStream(this.log);
         recordStream.on('error', err => {
-            this.logger.error('error receiving raft log',
+            this.log.error('error receiving raft log',
```

Comment:
Can we also update  to follow the same naming scheme to help avoid this in the future with this?

https://github.com/scality/backbeat/blob/6ec8bd753cc8843305f53879f768e4d28092ebe0/lib/queuePopulator/IngestionProducer.js#L18 

There may be some magic with setting that for the inherited `stream.Transform`, but maybe we could just remove that in the `ListRecordStream` constructor altogether.

Reply:
I think keeping the 2 different can help clarify that they are in different classes.

This is code copied from Arsenal and will be rf'ed in the future, at the moment if it's not breaking anything I think it's better to leave as is.